packages/flutter_tools/lib/src/commands/fuchsia_reload.dart

```diff
@@ -343,9 +363,16 @@ class FuchsiaReloadCommand extends FlutterCommand {
     return <String>[path, name];
   }
 
+  Future<List<_PortForwarder>> _forwardPorts(List<int> remotePorts) {
+    final String config = '$_fuchsiaRoot/out/$_buildType/ssh-keys/ssh_config';
```

Comment:
As discussed offline, the configuration here seems to create the tunnel as a deamon that outlives the flutter tool command.

Reply:
Done